src/App.js

```diff
@@ -1,26 +1,24 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import Header from "./component/Header";
+import Home from "./component/Home";
+import Restaurants from "./component/Restaurants";
+import OrderHistory from "./component/OrderHistory";
+
 
 class App extends Component {
```

Comment:
why is this a class instead of a function? Also, in all the rest, you use `export default <definition>`, but in this one you split the definition from the export.

Reply:
The App.js component is generated by create-react-app. I was just trying to minimize the number of changes that would need to be documented or that a student would need to make.

Reply:
Proptypes are on the todo list. I'll change the file extensions and default exports.

Reply:
TBH, I wish they would update CRA to use functional components since that's the direction they are going.

In any case, I think we should do this the "right way", even if the guide won't specify to do it.